### PR TITLE
Added tests for performance requirements

### PR DESCRIPTION
**Pull request by akshitg3**
#199 
Added tests for performance requirements

### CODE REVIEW

File: docs/VnVPlan/VnVPlan.tex [hunk 1]
```diff
@@ -900,18 +1045,18 @@ \subsection{Traceability Between Test Cases and Requirements}
   UH-UP1 & \\ \hline
   UH-UP2 & \\ \hline
   UH-AS1 & \\ \hline
-  PR-SL1 & \\ \hline
+  PR-SL1 & TC-PR-1\\ \hline
   PR-SL2 & \\ \hline
```
Comment (kyle4908) [inline docs/VnVPlan/VnVPlan.tex:1049]: this requirement can be removed from the SRS, so a strikeout of it in the SRS like Rafeed did earlier https://github.com/Spitgranger/SyncMaster/pull/221

File: docs/VnVPlan/VnVPlan.tex [hunk 2]
```diff
@@ -900,18 +1045,18 @@ \subsection{Traceability Between Test Cases and Requirements}
...
   PR-SL3 & \\ \hline
-  PR-SC1 & \\ \hline
+  PR-SC1 & TC-PR-2\\ \hline
   PR-SC2 & \\ \hline
-  PR-PA1 & \\ \hline
+  PR-PA1 & TC-PR-3\\ \hline
   PR-PA2 & \\ \hline
```
Comment (kyle4908) [inline docs/VnVPlan/VnVPlan.tex:1054]: I guess maybe a test of uploading a certain amount of documents and then viewing them in the system and making sure that over 99.99% of them can be viewed correctly and do not have any corruption

File: docs/VnVPlan/VnVPlan.tex [hunk 3]
```diff
@@ -900,18 +1045,18 @@ \subsection{Traceability Between Test Cases and Requirements}
...
-  PR-CR1 & \\ \hline
-  PR-CR2 & \\ \hline
-  PR-SE1 & \\ \hline
+  PR-RFT1 & TC-PR-4\\ \hline
+  PR-RFT2 & TC-PR-5\\ \hline
+  PR-CR1 & TC-PR-6\\ \hline
+  PR-CR2 & TC-PR-7\\ \hline
+  PR-SE1 & TC-PR-8\\ \hline
   PR-SE2 & \\ \hline
   PR-LR1 & \\ \hline
```
Comment (kyle4908) [inline docs/VnVPlan/VnVPlan.tex:1061]: this is a bit difficult to verify, but I guess we can just include a vague thing explaining how development team will evaluate the modularity of the system

Comment (Spitgranger) [reply docs/VnVPlan/VnVPlan.tex:1061]: can say things such as ensuring testability (i.e. actual unit can be tested and the system isn't tightly coupled) and ensuring changing x part of the system doesn't affect y

**Review by kyle4908 — APPROVED** [review]

**Pull request merged**